### ISSUE TITLE
Some way to check if app is running locally

### ISSUE THREAD

**Issue by brc-dd** [lagonapp/lagon#786]
**Is your feature request related to a problem? Please describe.**

If I run `lagon dev app.ts` or `NODE_ENV=development lagon dev app.ts`, even then the `NODE_ENV` remains `production`.  Is there any way as a library author we can check if someone is running it locally or in prod?

My code:

```ts
console.log(process.env.NODE_ENV)

export function handler(req: Request) {
  return new Response('Hello world')
}
```

<img width="479" alt="image" src="https://user-images.githubusercontent.com/40380293/233843057-7eae7ed9-c802-41ad-b8ae-b7e9cedc0166.png">

**Describe the solution you'd like**

Set some env variable to distinguish between `development` and `production`

**Describe alternatives you've considered**

From inside the handler I can do something like `request.headers.get('x-lagon-region') === 'local'`

**Additional context**
N/A

**Comment by QuiiBz** [comment]
Agree on this feature, we should set `process.env.NODE_ENV` to `development` when using `lagon dev`, and `production` (the current behavior) when deploying.

The code that sets it is here:
https://github.com/lagonapp/lagon/blob/04fdacd7b2f12df08d8dec2e8e0d27b1d94a66b1/crates/cli/src/utils/deployments.rs#L221

Would you like to make a PR?

**Comment by brc-dd** [comment]
> Would you like to make a PR?

Ah, I'm not much familiar with rust 😅 Also IG it's not simply just changing it to development, right? (as that code is being called before bundling for prod too).

**Comment by QuiiBz** [comment]
Yeah, it requires a bit more changes. Will do the PR myself, thanks!

**Comment by brc-dd** [comment]
Great, thanks for swift response! Also, are there any plans to support PBKDF2 for `crypto.subtle.deriveBits/deriveKey`?

**Comment by QuiiBz** [comment]
`SubtleCrypto.prototype.deriveBits` isn't supported at all right now, but we could add support for it. I'm logging an issue for it!
